### ISSUE TITLE
Failed to run T-Rex Runner v. 1.0 on Window 10 64-bits

### ISSUE THREAD

**Issue by ivan-enzhaev** [Yeti47/TrexGame#1]
Hi,

I tried to run [trexrunner_1-0](https://github.com/Yeti47/TrexGame/releases/tag/v1.0) but it does not run. I have Window 10 64-bit. I tried to run from command line but it does not write anything:

![image](https://user-images.githubusercontent.com/3908473/133087370-a232eaa5-4b09-48c7-a14f-3cde500b43a1.png)

I just see nothing.

**Comment by Yeti47** [comment]
Hi there, thanks for opening the issue. However, I cannot replicate the error. I ran the executable on two different Win10 64bit machines and everything seems to be working just fine. I'll see if I can find out what's causing the problem.

**Comment by Yeti47** [comment]
I just checked it on another machine that doesn't have the .NET Core runtime installed and in that case it didn't work for me either. So that will most likely be the problem. So there are two possible solutions:
1) You install the .NET Core runtime.
2) I create another "self-contained" release.

Let me actually go ahead and try to give the second option a shot.

**Comment by Yeti47** [comment]
Hi, again!
I just released a self-contained version of the app. It should now run without having the .NET Core runtime installed.

[Download it here](https://github.com/Yeti47/TrexGame/releases/tag/v.1.0_self-contained)

This version is larger in size because the required runtime assemblies are compiled into the exe, but it should now run on your machine. Let me know if the issue has been resolved, so that I can close it.
